.env.dev

```diff
@@ -0,0 +1,8 @@
+DEBUG=1
+SECRET_KEY='7*fsmud8l*&)o^m2jp4$fn%nk$g)xh9z$y+q96_uj_+vmvirdj'
+DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
+POSTGRES_DB=arcanis_dev
+POSTGRES_USER=arcanis_user
+POSTGRES_PASSWORD=arcanis_password
```

Comment:
same comment here-- what's the difference between this .env and the other one

Reply:
Just for local dev -- one is actually for running the server in docker, and the other is for running the server on the local host machine. These might change a lot by the time I'm done, so I haven't thought about whether I want to keep both yet tbh.